tests/software/bowtie/align/test.yml

```diff
@@ -42,6 +42,6 @@
     - path: output/bowtie/bowtie/NC_010473.rev.2.ebwt
       md5sum: f3c398bba5158f4039334a932d79c051
     - path: output/bowtie/test.out
-      md5sum: a23e9a2a76e949aeb3693bcfae41a615
+      md5sum: a81cb18024616415a6cec3108a36fccd
```

Comment:
Any idea why this changed?

Reply:
```
# reads processed: 10000
# reads with at least one alignment: 2 (0.02%)
# reads that failed to align: 9998 (99.98%)
Reported 2 alignments
```

This is the latest version which is only showing 2 alignments...are you able to run `pytest` via the old tests so we can see the difference?

Reply:
The old version generated the output below which is why the tests are failing:

```
# reads processed: 10000
# reads with at least one alignment: 1 (0.01%)
# reads that failed to align: 9999 (99.99%)
Reported 1 paired-end alignments
```

I guess this highlights the importance of using seed values where possible in `main.nf` for the tests to always generate the same alignments and not by random. At least, this is what I think is happening here.